clickstream/docs/index.rst

```diff
@@ -63,7 +72,17 @@ are quite large and depending on your network connection may take
         ksqldb-server     bash -c # Manually install ...   Up      0.0.0.0:8083->8083/tcp, 0.0.0.0:8088->8088/tcp
         schema-registry   /etc/confluent/docker/run        Up      8081/tcp                                      
         tools             /bin/bash                        Up                                                    
-        zookeeper         /etc/confluent/docker/run        Up      2181/tcp, 2888/tcp, 3888/tcp    
+        zookeeper         /etc/confluent/docker/run        Up      2181/tcp, 2888/tcp, 3888/tcp   
+
+
+#. Once you've confirmed everything is running in previous step, run the source connectors via ksqldb-cli 
+
+   .. code:: bash
+
+        docker-compose exec ksqldb-cli bash -c "ksql http://ksqldb-server:8088 <<EOF
+        run script '/scripts/create-connectors.sql';
+        exit ;
+        EOF"
```

Comment:
Once the connectors are created, how about showing a screenshot of them running in C3?

Reply:
There's [an image for that already](http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/4375/5.5.0/tutorials/examples/clickstream/docs/index.html#verify-the-data) should I move it up to the `Create the Clickstream Data`?

I'm fine with where it goes either way.

Reply:
EDIT: Looking at the new page, I think it makes sense to move the C3 Connector image up to this section